libs/api_common/device_helper.cpp

```diff
@@ -129,6 +129,7 @@ register_wait_async_ioctl_operation(_Inout_ async_ioctl_completion_t* async_ioct
 
     // Set the event on the thread-pool wait object.
     ebpf_assert(async_ioctl_completion->wait != nullptr);
+    _Analysis_assume_(async_ioctl_completion->wait != nullptr);
```

Comment:
Can we combine ebpf_assert and _Analysis_assume_ into the same macro rather than requiring both everywhere?

Reply:
Files this issue to track this suggestion. https://github.com/microsoft/ebpf-for-windows/issues/1584